WORKSPACE

```diff
@@ -46,21 +46,19 @@ rules_proto_toolchains()
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "69de5c704a05ff37862f7e0f5534d4f479418afc21806c887db544a316f3cb6b",
+    sha256 = "2b1641428dff9018f9e85c0384f03ec6c10660d935b750e3fa1492a281a53b0f",
     urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.27.0/rules_go-v0.27.0.tar.gz",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.27.0/rules_go-v0.27.0.tar.gz",
-        "https://storage.googleapis.com/builddeps/69de5c704a05ff37862f7e0f5534d4f479418afc21806c887db544a316f3cb6b",
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.29.0/rules_go-v0.29.0.zip",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.29.0/rules_go-v0.29.0.zip",
```

Comment:
I see that there isn't a mirror on google storage (like there used to be before your changes).
Please see last bullet in the [doc for updating golang](https://github.com/kubevirt/kubevirt/blob/main/docs/update-go-version.md) that leads to [this guide](https://github.com/kubevirt/project-infra/blob/main/robots/cmd/uploader/README.md).

Reply:
the guide is a little bit outdated. The mirroring is done by our bot after the PR gets merged. It will create a follow-up PR automatically with the dependency mirrored. For instance: https://github.com/kubevirt/kubevirt/pull/6992

Reply:
Cool! good to know
I will make a PR to change my golang-bump guide